src/routes/widgets/handlers.js

```diff
@@ -24,8 +24,8 @@ export async function getWidget(req, res) {
     // This returns the query object with "region" property filtered by user permissions
     const query = await setReadRegions(req.query, req.session.userId, true);
 
-    // convert the query to scopes
-    const scopes = filtersToScopes(query);
+    // Determine what scopes we need.
+    const scopes = filtersToScopes(query, modelType);
```

Comment:
I think I finally see where `modelType` is useful (thanks for bearing with me, and the following is mostly for me thinking out loud!). Widgets were originally writing thinking the backing data would always be `ActivityReports`. There are now new widgets that are backed by data from the `Grants` table. Just a heads up, there may be other assumptions baked into widgets/filters so be on the lookout.